views/edit_or_add_mixin.pug

```diff
@@ -34,13 +34,22 @@ mixin editScript(children, chorals, thisChoral)
       displayChildren()
     })
 
+    $("#mainForm").submit(() => {
+      if($("#childChoral").val()){
+        return confirm("You have not added the child selected to the choral, should we continue?")
```

Comment:
I suggest that we remove the "Add Child" button, and go with an on click item and add the child when the user clicks on the child name